lib/tapioca/commands/abstract_gem.rb

```diff
@@ -77,7 +77,7 @@ def initialize(
       def gems_to_generate(gem_names)
         return @bundle.dependencies if gem_names.empty?
 
-        gem_names.each_with_object([]) do |gem_name, gems|
+        (gem_names - @exclude).each_with_object([]) do |gem_name, gems|
```

Comment:
Yeah if we could get the test working without doing `gem foo --exclude foo` we could keep the error I think. But not a big deal.

Reply:
I'm not sure what that test is intended to verify. If you have any suggestions, I'm happy to take those into account. Otherwise, I'll just leave this as-is.